packages/utils/src/Asserts.tsx

```diff
@@ -1,4 +1,5 @@
-export default function assertNotNull<T>(
+// eslint-disable-next-line import/prefer-default-export
+export function assertNotNull<T>(
```

Comment:
This can just be a `.ts` file instead of a `.tsx` file, as it does not contain any React/JSX